### DISCUSSION TITLE
"Aether scalar tensor theory confronted with weak lensing data at small accelerations"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#128] [Gravitation Models]
Just published in A&A, "Aether scalar tensor theory confronted with weak lensing data at small accelerations" by
T. Mistele, S. McGaugh and S. Hossenfelder at [https://arxiv.org/abs/2301.03499](https://arxiv.org/abs/2301.03499)

  This model is a "relativistic field theory for MOND", an upgrade from the classical Milgromian MOND.

  The number of epicycles in this model are obtained from the number of free parameters, new hypotheses, and new equations added to (or replacing) accepted physics - Newtonian and general relativity.

1- A new hypothesis, a 'new ingredient':
 "An important ingredient in the AeST model is a so-called ghost condensate"

2- A new hypothesis, a 'new field $\phi$' added to Newton's gravitational field:
 "the model can be described by two fields, $\Phi$ " [a Newtonian potential] "and $\phi$, both fields are sourced by the baryonic energy density $\rho_b$ and the ghost condensate density $\rho_c$"

3- A new equation, the 'equation of motion for the new field $\phi$':
 "whose equations of motion are" [Poisson equation for $\Phi$ ], [MOND equation function of $\Phi$, $\phi$ and $f_G$] "(Eq. 1b)"

4- A new parameter, ' $f_G$ ':
 " $f_G$ is the conversion factor between Newton's gravitational constant $G_N$ and the constant $G$ that appears in the Lagrangian"

5- A new equation, 'the interpolation function for MOND':
 "The function $\mu$ can be freely chosen in this model and corresponds to an interpolation function of MOND"

6- A new equation, the 'ghost condensate energy':
 "The ghost condensate energy density $\rho_c$ is given by" [equation function of $\Phi$, $\phi$, $M$ and $Q_0$] "(Eq. 2)"

7- A new parameter, ' $m$ ':
8- A new parameter, ' $Q_0$ ':
 "where $m$ and $Q_0$ are constants"

  With EIGHT epicycles (ignoring the Lagrangian in the appendices) it fits 1-galaxies, 2-galactic clusters, 3-fluctations in the CMB, 4-the matter power spectrum, and has a tensor mode that 5-propagates at the speed of light.

  Impressive work done by Mistele, McGaugh, and Hossenfelder, but given the number of new elements required to get this model to work, this looks more like 'function fitting' than a fundamental model of gravity.

**Comment by RedshiftDrift** [comment]
"Strong constraints on the gravitational law from Gaia DR3 wide binaries" [https://arxiv.org/abs/2311.03436](https://arxiv.org/abs/2311.03436)
<<... we find that... fully consistent with Newtonian gravity but ***excludes MOND at 16-sigma confidence*** [...] concluding that MOND must be substantially modified on small scales>>

There seems to be this general trait shared by astrophysicists which makes them modify failed hypotheses (i.e. add epicycles) rather than reject them.  Why?

**Reply by budrap00** [reply]
Well, it "worked" for Newtonian Dynamics and General Relativity, so why not! The real problem is the hubristic conceit that every time a mathematical model is found that fits a certain phenomenon it is declared a "universal" law and that designation is treated as irrevocable. When the formalism subsequently fails under different conditions either the law or more often reality itself has to be modified to maintain the "universal" delusion. 

In the case of MOND, ND, and GR, all three are math models describing gravitational effects on certain scales. What those models do not describe is the cause of the observed and measured effects. Consequently, when the underlying causal system is altered the models cannot reflect the new circumstances because they do not model the causal system. This shouldn't be that surprising but it is because of the mathematicist framework that contemporary theoretical physics operates under. 

In this case it is a variant of mathematicism called instrumentalism wherein the instrumentalist holds that as long as a math model gives accurate results nothing more need be known about the underlying system. That belief has been falsified repeatedly with respect to gravity but axiomatic beliefs are impermeable to factual refutation, and so the epicycles roll merrily along.

**Reply by HanDeBruijn** [reply]
@budrap00 :
>What those models do not describe is the cause of the observed and measured effects.
[ .. ] more need be known about the underlying system

1. Please **define** what a **cause** is.
2. How deep must we go to know more about your _underlying system_ ??

**Reply by budrap00** [reply]
1. Cause: The producer of an effect, result, or consequence. (The American Heritage® Dictionary of the English Language, 5th Edition)
2. It is not a question of going deep, it is a question of paying more attention to physical reality than to abstract models that only converge with reality at their output. If a mathematical mechanism that achieves such convergence requires a host of undetectable effluvia (to achieve that convergence) it is useless as a scientific guide to the nature (as in inherent character or basic constitution) of physical reality.

**Reply by RedshiftDrift** [reply]
1. Cause of gravity: mass.
2. "underlying system", "reality": metaphysical ideas outside the realm of science.
3. "astrophysicists modify failed hypotheses, why?" because they can't find new ideas but need to keep a living income.

**Reply by budrap00** [reply]
1. That gravity is associated with mass has been known since Newton but Newton did not consider "instantaneous action at a distance" to be an adequate description of the gravitational mechanism (which quite obviously extends beyond a gravitating body) nor did Einstein consider geometrized spacetime to be a correct description of the gravitational mechanism. As of this point the gravitational mechanism is unknown.
2. "Underlying causal system" refers to the unknown gravitational mechanism - it is not a metaphysical concept; it is an open scientific question - except to instrumentalists who are satisfied with some math that only works well on solar system scales. Physical reality (that which can be directly observed, measured and detected) is the only proper subject of science. Math is a tool of science.
3. The economic conditions of the scientific academy would not be an issue if there were no unassailable, unscientific, axiomatic truths and scientifically baseless philosophical conceits (mathematicism, instrumentalism) constraining theoretical and observational physics.

**Reply by Francois-Zinserling** [reply]
NASA: _gravity: an invisible force that pulls objects toward each other._ 
Admittedly it is a public website and needn't be very scientific.

My opinion:
Cause of gravity: Curved space, gradient of space, vacuum density, (insert your favourite term for aether here)
Player in the game: mass

Mass is of course instrumental but only indirectly contributes to gravity. When you feel the effects of gravity lightyears away, the mass is not causing that interaction. OK, it **caused** it, yes, but right now you are moved by the aether.

**Comment by genotics** [comment]
In the essay collection Einstein and Poincare: the physical vacuum, edited by V. Dvoeglazov in 2006, there appeared the last article written by JP Vigier (posthumously), sent to me in a rough draft before his death,  and the last article written by Adolphe Martin, who died in 2008. 
Vigier's abstract reads: "In this work we analyze new possible physical properties of extended Quantum particles considered to be sets of vortices built (with their associated real “pilot” waves) from a tight combination of localized “holes” (gravitational) and “bumps” (electromagnetic) propagating within a moving, real, stochastic physical Dirac-type “aether” within an infinite three-dimensional flat-space. In this model the corresponding interaction forces are mediated by new vector bosons." 
However, in my opinion he does not succeed in showing how the holes and bumps can be formed from the stochastic aether.
Martin also presupposes a stochastic medium, a cosmonic gas, and proposes that "Newtonian gravitation arises from the Clausius term added to the ideal gas equation to account for the space occupied by cosmons, fundamental particles of cosmonic gas. The gravitational force is explained as a pressure due to static pressure and cosmon concentration." 
While both these approaches, which appear to me complementary up to a point, may perhaps fall short of achieving a unified account of the forces, they nevertheless suggest a paradigm free from the myriads of epicycles to which the instrumentalists are forced to resort, and deserve consideration in the context of the renewed campaign to marry Newtonian and relativistic gravity.

**Comment by bgreenman** [comment]
Excluding MOND at 16-Sigma confidence, inability to find a medium (quantum vacuum, other dimensions, aether,etc....) is an indication mainstream physics theorists are having trouble linking both Newtonian and Einsteinian gravity concepts in a coherent gravity theory

**Reply by RedshiftDrift** [reply]
They also excluded Newtonian gravity to 10 $\sigma$.  This has nothing to do with gravity theory but instead reflects the difficulty of data analysis to high accuracies in astronomy.
Their uncertainties are simply under-estimated by a factor of ten.

**Reply by bgreenman** [reply]
correct

**Comment by RedshiftDrift** [comment]
"Gravity might be a force after all" Hossenfelder https://youtu.be/MA75uYUF5oc
This gets rid of the mathematical $g_{\mu\nu}$-type "space curvature" and proposes a *physical mechanism* of particle interactions (like photons, not "Le Sage"-type) to describe gravity.
Much better than GR😛, and Sabine approves of this! 😉

**Reply by budrap00** [reply]
From Sabine's account:

>Space is flat, always has been, always will be. Instead, they introduce four new bosons, one for each direction of space and time.
>
>... the four bosons merge, and give exactly a spin-two particle which is the graviton.

So four new imaginary, and doubtless undetectable, particles are introduced into an imaginary and undetectable flat Space where they combine to form an old imaginary and undetectable particle, the graviton. It sounds like just another mathematicist fairy tale to me; one that, as usual, doesn't correlate with anything observable in physical reality. Sabine has her own mathematical reservations:

>Gravity in flat space, but with four gauge bosons, a matrix field to hold phase factors, and torsion instead of curvature? Even if it works, good luck convincing anyone to use it. Then again, it’s not like string theory is exactly known for its parsimony and they could get some thousands of physicists to swallow that.

Bottom line - this is just some math, not physics. Even if it "works" in the Ptolemaic sense, it's a scientifically meaningless nothingburger.

**Reply by RedshiftDrift** [reply]
I agree, it only "works" in the Ptolemaic sense, but this offers a guide to discover observable effects behind gravity instead of denying that gravity is a force because it is a result of space distortion.

**Reply by budrap00** [reply]
I have a problem with the way you are using the term "force". To my way of thinking force describes a certain type of effect that arises in matter-energy interactions. The concept of force as it is commonly used in physics represents another reification of a relational concept into a physical entity which consequently renders an effect as its own cause. Saying gravity is a force doesn't mean anything because it does not describe the physical mechanism by which the force is affected.

A golf club is not a force but it can used to exert a force on a golf ball. Electromagnetic radiation is not a force but it can exert a force on matter in an encounter. A force in this approach is not a causally-interacting substance in the exactly same way that spacetime is not a causally-interacting substance. They are both only relational concepts - the reified versions have no empirical evidence supporting their existence. Those reified concepts are just additional members of the very large zoo of invisible things that is such a prominent and damning feature of modern theoretical physics.

**Reply by RedshiftDrift** [reply]
It's just a matter of language... Physicists label the different types of forces.  The only way to detect gravity is through the effect it has on matter, which appears as a force that makes matter accelerate, hence the "force of gravity".  The only way to detect electric fields is through the effect it has on charges, which appears as a force that makes charges accelerate, hence the "electrical force".

I am not claiming that gravity, and electricity are "substances".  Science only describes effects in terms of other effects, until we reach the *limit* of the most fundamental effects.  We don't know how to describe these *'most fundamantal effects'* in terms of even more fundamental effects.  Then it's metaphysics, with "substance", "cause", "effect", etc.

 Nobody knows the underlying physical mechanisms behind gravity and electricity.  But for gravity, I reject the description that uses "space distortion".

**Reply by budrap00** [reply]
I'll take a pass on the semantical discussion and cut to the physics of the matter.

"Nobody knows the underlying physical mechanisms behind gravity and electricity."

The case can be made that this is because no attempt is being made to understand the nature of that mechanism. This neglect or indifference is particularly striking given that a correlation between gravitational effects and electromagnetism has been [known for a century](https://en.wikipedia.org/wiki/Gravitoelectromagnetism). The math for both phenomena is quite similar. Observationally, both gravitational and electric "field effects" arise in the context of an aggregate electromagnetic field.

Large gravitating bodies (stars and galaxies) are also copious emitters of electromagnetic radiation. The radiation density of those emitters falls off as 1/r^2 which is exactly the same rate at which the strength of the invisible gravitational field falls off.

Everywhere we look in the Cosmos there is a field of [Ambient Electromagnetic Radiation](https://journals.sagepub.com/doi/10.1177/0003702818767133) that is present wherever scientists imagine there to be spacetime. Imaginary spacetime is in all of the cosmological models I have ever encountered but the AER is not.

The AER has been present in every laboratory where every quantum or electrical experiment has ever been run but quantum models have a flat, static, 4-dimensional Minkowski spacetime and no AER. The exception to that rule is Bohmian Mechanics which has a mathematical analog of the AER in its chaotic quantum subfield. BM is disfavored by the Modern Theoretical Physics community, apparently for the unforgivable sin of making sense.

Electromagnetic radiation is an observed physical phenomenon - it is not an effect, it is a causal agent. It provides the very basis for our concept of observation. Every scientist knows about the AER on some level. The AER permeates the room you are sitting in, though it won't necessarily have the same frequency distribution as the cosmic AER. The AER is everywhere there is no matter but it is not present in either of the standard theoretical models of MTP. Those models are instead full of things that are not there. Those two facts are correlated.

**Reply by RedshiftDrift** [reply]
@budrap00 
> The case can be made that this is because no attempt is being made to understand the nature of that mechanism. [...] Observationally, both gravitational and electric "field effects" arise in the context of an aggregate electromagnetic field.

F.Y.I. many physicists have made attempts at understanding the nature of gravity and electricity, including myself and fellow students (when we were in undergrad 40 years ago).

The reason nobody knows the underlying physical mechanisms behind gravity and electricity is because nobody has found an experimental way to validate one idea and eliminate the bad ones.  Example: what experiment can you do that will tell us who is right?

"...an aggregate electromagnetic field.  Large gravitating bodies are copious emitters of electromagnetic radiation. The radiation density of those emitters falls off as 1/r^2" - [@budrap00]

or

"*Positive-positive and negative-negative repulsion* does not exactly cancel *positive-negative attraction*; what's left is a very small attraction that falls off as 1/r^2 but is orders of magnitude smaller and appears as gravity." - [@RedshiftDrift and colleagues, 1983]

There is no way to tell them apart because they make the same predictions.

**Reply by budrap00** [reply]
The 1983 analysis in which you claim a net attraction based an electrostatic analysis does not seem plausible because the attraction would apply equally positive, negative and neutral matter; maybe it works in the math but the physics is a little fuzzy to say the least. 

Regardless, that analysis has nothing to do with the _fact_ that there exists an Ambient Electromagnetic Radiation field. The density of the AER around a radiating body falls off as 1/r^2 in the exact same manner as the  gravitational effect. 

You don't seem to want to dispute that _fact_ but you have just gone out of your way to avoid it in your response by offering an old and discarded analysis as a way of not addressing the point raised. The question I have for you then is simple. 

Why? Do you have a specific scientific justification for ignoring the AER that you and every other material body exists in? It is empirically, what is there after all, not spacetime, the aether, gravitons, or whatever invisible friend du jour props up the latest theoretical explanation for gravity on offer here. How do you justify ignoring the AER?

**Reply by HanDeBruijn** [reply]
@budrap00 .
>The density of the AER around a radiating body falls off as 1/r^2 in the exact same manner as the gravitational effect.

Huh? The **Ambient** Electromagnetic Radiation - around a radiating body ? - that **falls off** as ($1/r^2$ ) ?

**Reply by budrap00** [reply]
@HanDeBruijn I could have worded that better, thanks for pointing it out. The emitted radiation density falls off as 1/r^2 of course but how does that affect the AER? The proposal is that it is the Emitted Radiation-density Gradient that causes the observed gravitational effects. The observed gravitational effect on starlight passing close by the emitting body is that it is curved and slowed while travelling through the ERG. (Actually the curvature is observed for starlight and the slow down for radio signals.) So the AER should become denser in direct response to the ERG in the vicinity of the radiating/gravitating body.

That behavior of the AER is not directly observed except for the visible starlight component and radio signals but it is not unreasonable to think that the entire AER spectrum is similarly affected. This density gradient would represent only a minimal part of the emitter's overall ERG. Since the ERG falls off as 1/r^2 so should its effect on the AER.

Addendum: It is generally noted that electromagnetic radiation traversing a gravitational field behaves as it does in a medium with a density gradient. All this proposal does is to identify that medium as the emitted radiation from the "gravitating" body.

**Reply by Francois-Zinserling** [reply]
It is known that light curves toward a denser medium. This is known for matter e.g. glass, water and other transparent substances. The more density, the more the curving. 
Now - by extension - the aether (or let's keep your acronym AER for now) must be the opposite of 'matter'. In other words, more dense matter means less dense AER. 
This also allows e.g. explanations of why light slows down inside dense matter - more matter, less AER and thus slower light ... and slower clocks. .. I digressed a bit.

My next observation is that the curvature of light or radio signals is not caused by gravity, but caused by the same effect that causes gravity. I think this is in line with what you are saying @budrap00 . IOW - from my side - less dense AER, more curvature. Now this implies less dense AER 'around' matter and for this an accepted explanation is lacking. My attempts have not gone far to get accepted although I have not seen any better explanations for the mechanics of gravity. In simple terms, this less dense AER would then gradually become 'more dense' at a ratio of $1/r^2$ which then provides the mechanics for a gravity gradient toward the central mass .. well ... around every mass.

I am also firmly of the belief that the AER is aptly termed, since my model is also based on photons / electromagnetic radiation. What I stand firm on is that gravity cannot be caused by conventional (or leftover, or background) radiation, because giant emitters e.g. stars, would have been anti-gravity machines. This is not observed. So we need a different (yet also not widely accepted currently) mode for the AER such as the photon pairs proposed by Anilla et al. These photon pairs would penetrate through matter and are proposed to be only 'broken' if a direct encounter with matter ensues. 
It remains to find proof of these photon pairs, and possibly to propose a mechanism where they can be regenerated from the real AER. My model is therefore based on the equivalent of 'your' AER being the saturation of space with photon pairs.

**Reply by HanDeBruijn** [reply]
@budrap00 .
>the standard theoretical models of MTP.

What is MTP ?

**Reply by HanDeBruijn** [reply]
@budrap00 , @Francois-Zinserling .

Perhaps of interest to both of you: [Variable speed of light. Dicke's proposal_(1957)](https://en.wikipedia.org/wiki/Variable_speed_of_light#Dicke's_proposal_(1957)) .

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn that almost makes more sense already than the entirety of the past century of 'Standard Model'. Thank you for the link.

I don't prescribe to Mach's principle (as a cause of gravity) for lack of causality - but if the mass of the universe 'appears' to increase, it will make the horizon clocks 'appear' to run slower (relative to our clock) and voila, the horizon is redshifted. 
Maybe a bit simplistic explained, but at least it doesn't require an expansion.

**Reply by budrap00** [reply]
@Francois-Zinserling The term Ambient Electromagnetic Radiation refers to observed phenomena. There is in any location you might choose to identify some amount of omnidirectionally sourced electromagnetic radiation - this is not a theory, it is a fact of existence. Your misappropriation of the acronym "AER" in the context of your model, where you piggy- back hypothetical photon pairs onto the term, means that the "AER" of your model is incongruent with the Ambient Electromagnetic Radiation that is actually there. 

The point of my discussions here is that it is possible to account for empirical observations such as the gravitational effect by properly accounting for the behavioral interactions of empirically observable physical phenomenon such as material bodies and electromagnetic radiation - without recourse to invisible, undetectable hypotheses. In the specific case of gravitational effects there is observational and theoretical evidence that those effects arise in the context of matter-electromagnetic radiation interactions:

1) For massive radiating and gravitating body (stars and galaxies) the radiation density gradient and gravitational effect both fall off as 1/r^2.
2) Light passing in the vicinity of a star or galaxy behaves as if it were traversing a medium with a density gradient. The emitted radiation surrounding those bodies constitutes just such a medium.
3) As I've pointed out previously, you can apply some General Relativity based math for gravitational redshift to an expanding spherical wavefront of light emitted by a galaxy and generate a redshift similar to that of the standard model. Since the SM redshift seems unrealistic this is of limited interest. More important is that the observed energy loss (redshift) of an expanding spherical wavefront can be physically attributed to its gradual absorption by the intervening matter. This again accounts for a "gravitational effect" as being produced by the interaction of matter and electromagnetic radiation.

None of the foregoing constitutes a "model". It is a recitation of certain facts suggesting that gravity is not a separate force or interaction but an effect of matter-energy interactions. Could that be the basis for a realistic model of gravity? It could be but that would require theorists to pay closer attention to the facts of existence rather than to the landscapes of their febrile imaginations.

**Reply by Francois-Zinserling** [reply]
@budrap00 
Yes, my bad. I shall refrain from using AER.
The aether cannot be remnant electromagnetic radiation. It cannot be limited to what we currently know. It will be news when we get evidence of it. Yet it is ambient and it must be a (special) mode of electromagnetic radiation.

You are correct that we should base our hypotheses on facts, and build from there. Then you may be quick to recognise that we do not know of any electromagnetic radiation that can 'pull' on an object. It can only transfer momentum in the direction of travel - iow PUSH - on an object. 
1. This leaves a problem with your high density of AER around masses, because this should provide an **antigravity** mechanism which gets stronger closer to the mass as to $1/r^2$. This is not observed.
Your idea that there is a relation is not insane, yet now you have to invent something new to make this work, but you have to stick to your rule of 'working with what is known'. See the problem here?
2. As I have mentioned, I am firmly of the opinion that curved light is not caused by gravity, but caused by the same mechanism that causes gravity. We seem still aligned here.

Do you have a proposed mechanism how the $1/r^2$ ER from one mass can attract other masses?

**Reply by budrap00** [reply]
@Francois-Zinserling First let me emphasize that in the vicinity of a radiating body (star or galaxy) there are two distinct radiational components: 
1) the AER that surrounds all free standing material bodies - that is "The Spectrum of the Universe" as described in the document of the same name. This radiation does not have a density gradient but is approximately uniform in distribution around the body.
2) the radiation being emitted omnidirectionally by the body. That radiation has a gradient, the Emitted Radiation Gradient. It is that ERG that comprises the medium through which the AER travels and is observed to curve, as electromagnetic radiation does in any medium with a density gradient. This fully accounts for that observed "gravitational effect" - without gravity.

I want to stress that I am not proposing a one-size-fits-all electromagnetic radiation mechanism for all "gravitational" effects, only that how matter and electromagnetic radiation interact under different conditions can account for the various effects that are now attributed to "gravity". That is the point of the expanding spherical wavefront of electromagnetic radiation example - an observed effect that can be modeled using gravitational math can be ultimately attributed to the interaction between the wavefront and the intervening material bodies it encounters during expansion.

As far as the apparent gravitational "pull" between two bodies goes it can be described as involving both the AER that lies between them and their interaction with each other's ERG. In this conception the ERG _is_ the gravitational field and the AER _is_ the inertial field.  

As is known the AER is curved by the ERG in the vicinity of each body and the closest AER is curved onto and absorbed by each body. So the two bodies are pulling on the AER that separates them. Like two people pulling hand over fist on a rope, they aren't pulling on each other, only on the medium (rope, AER) that separates them producing what looks like an attraction but is only a mechanical shortening of the medium between them.

As the two bodies pass in each other's ERG they will follow a curved path in response to the ERG because the surfaces closest to each other are slowed in the denser portion of the ERG (clocks, all physical processes are running slower) than on the opposite surfaces farther up in the lower density "gravitational potential." The Emitted Radiation Gradient is the physical cause of the effects we attribute to an otherwise invisible gravitational field.

**Reply by HanDeBruijn** [reply]
![shut_up](https://github.com/a-cosmology-group/acg/assets/135433222/131488ca-6c19-4257-9646-5d6379472ce4)

**Reply by Francois-Zinserling** [reply]
Please note I'm commenting ... not arguing. We need to keep tugging at threads until we unravel all of it.
Despite being on this group, I am new to cosmology. Your spherical wavefront still interests me. There is something else in there to unravel in due time. 

I am also with you that the curve of light around massive objects is an ER effect and not a gravity effect. I don't think due to the ERG though, because cold vs hot central masses would have shown a noticeable difference in the bending of light and thus also in the mass/gravity of objects. We have no indication that black body spectrum is a measure of gravity or that the strength of an ERG affects the light curving. (so there's something else there that behaves a bit like ERG, except, not ERG)

Let's continue to hypothesise because there is a logic to your AER/ERG argument. Light passing a massive object curves toward the object. ERG is emitted radially outward and thus diminishes at $1/r^2$. A curve exists when (your) AER passes at the normal to the ERG, and the 'strength' of the curve is affected by the same $1/r^2$ ratio depending on the distance it passes the object. This works. Then one could argue that light that would barely have passed over the surface will now be curved 'into' the object. This provides you with a possible mechanism for gravity. One could argue that it doesn't absorb any light if it's not a black hole, only curves the path, but then it becomes more difficult to explain a net 'down' vector to cause gravity.

> So the two bodies are pulling on the AER that separates them.

Or being pulled by...? What you describe is a 'vacuum of sorts' of the AER between the masses. 
Welcome to push-gravity :) You are now in the halls of the enlightened. (just kidding, no, not kidding). "you can check out any time you like, but you can never leave"

But along with this comes a big problem - absorption. This one is a death-blow. It has been shown by many (Maxwell, La Place, Poincare, Feynman and others) that the absorbed energy will roast earth in an instant. 
For this reason AER, as a concept alone, cannot work and mainly due to this same reason the scientific community have ditched push-gravity entirely. 

But if we can find or propose a mode of EMR that penetrates matter without heating it - and interacts, then exits as a different mode - this topic can be saved. The exit mode would likely contribute to your AER but we still need to find or propose something new for the entry mode that doesn't result in energy absorption. 
Enter Anilla et al.

**Reply by budrap00** [reply]
@HanDeBruijn Shut up and calculate is the unthinking mantra of mathematicism. It is the reason MTP (Modern Theoretical Physics) is, scientifically speaking, a MESS (Meaningless Extrapolation of Senseless Superstitions). Mathematicism is one of the base superstitions of MTP.

**Reply by budrap00** [reply]
@Francois-Zinserling I would say we are having a debate, which is a civilized form of argumentation that sets aside the unnecessary emotional baggage that often accompanies "arguments". It is much appreciated.

I don't understand this point:

>One could argue that it doesn't absorb any light if it's not a black hole...

Everyday the Earth absorbs - and then reradiates - electromagnetic energy, so in what context does that argument hold? Consider for instance a blackbody the size and mass of the Sun in the IGM (Intergalactic Medium). At its surface it is absorbing all of the AER that is directly falling on it and then reradiating it as blackbody radiation such that the blackbody arrives at the condition of thermal equilibrium with the surrounding AER. Basic thermodynamics.

The BB radiation would be less dense at the surface than the radiation density at the surface of the Sun and therefore the density gradient would be less steep. That leads to the conclusion under the ERG=Gravity assumption that the blackbody would exhibit less of a "gravitational" effect than an equivalent star. Of course the only objects we observe with the size and mass of stars are stars so that theoretical prediction doesn't mean much. However...

>I don't think due to the ERG though, because cold vs hot central masses would have shown a noticeable difference in the bending of light and thus also in the mass/gravity of objects.

You seem to be basing your remark on an assumption that observations always show a light curvature in accord with theory. That is not the case. Gravitational lensing around a galaxy cluster needs considerable amounts of dark matter to "fit" the observations to the standard model. This might very well be a consequence of the fact that the x-ray hot plasma that constitutes up to 90% of a cluster's mass has a higher radiation output than an equivalent stellar mass would produce. That excess high energy radiation would be producing a stronger "gravitational" effect than the observed mass alone would predict.

Pushing gravity has been around for decades and it has always had the same problem, it requires the ubiquitous presence of some undetectable and peculiarly behaving substance to make it work. Like the two standard models it is a theory without the necessary physics to render it meaningful. That kind of theory-first approach to doing science over the last 50 years has been an unmitigated disaster. I don't understand why you find it (pushing gravity) so compelling.

**Reply by Francois-Zinserling** [reply]
@budrap00 
Apologies - I misrepresented this one.

> One could argue that it doesn't absorb any light if it's not a black hole...

I was only referring to 90' to R passing beams, which paths get curved by masses but only black holes can curve the light-path onto itself. Other masses merely 'bend' the paths of passing light. 

Of course EMR that is aimed direct will be absorbed and must be emitted to retain equilibrium.

For AER(in) and BB(out) there is an equilibrium, as you say. Energy in = energy out but so also momentum in = momentum out, no matter how the photons are transformed in the process. 

We have yet to find anything in physics that transfers a negative momentum, coordinate manipulations excluded. There is no working alternative besides push-gravity, there is only searching for the detail of its mechanics. When I 'tongue-in-cheek' welcomed you to push-gravity because of your proposed AER vacuum between masses, it remained for you to realise that the vacuum doesn't pull, but the AER positive pressure from outside pushes the masses toward each other.

That radiation(out) for stars affect gravity there is no doubt. Even here at earth distance from the sun the radiation push is significant. But it 'affects' on gravity does not align it to the cause of gravity. I doubt. As I've said before, if anything, stellar radiation would cause anti-gravity. So we need to look somewhere else for a solution, because here at earth the sun's attraction is still way more than it's radiation's push. 

> the ubiquitous presence of some undetectable and peculiarly behaving substance

is arguably the least of push-gravity's problems. It has been done this way before many times in physics, to propose something yet unknown; for instance photons which are very relevant to this topic. Notably there has been spectacular failures with e.g. dark matter, dark energy, and quarks, but that is why we continue to dig for real solutions.

**Reply by budrap00** [reply]
>Even here at earth distance from the sun the radiation push is significant.

That's not true at all. The radiation push from the Sun isn't even significant at the scale of a soccer ball. It has an affect on individual electrons; it is insignificant at the scale every day objects, let alone the Earth. All you're saying is that EMR does not cause pushing gravity. I'm not describing a pushing gravity caused by EMR momentum transfer, you are. 

What I am describing is the aggregate effect of the combined AER and ERG fields in the vicinity of a "gravitating" body as having an effect on the paths of both EMR and matter passing through it. There is no pushing effect being invoked or needed. Which brings me to this assertion:

>There is no working alternative besides push-gravity, there is only searching for the detail of its mechanics.

An alternative model is exactly what we are discussing here. Your opinion that there is no alternative to push-gravity is just that, an opinion. It is not supported by the evidence. The curved spacetime account of gravity is in a sense more robust though it also rests on an imaginary mechanism. At least the geometric math of GR is suggestive of such a mechanism, though Einstein did not agree with that interpretation.

>It has been done this way before many times in physics, to propose something yet unknown; for instance photons which are very relevant to this topic.

Well the photon concept is not relevant to cosmological scale phenomena. The photon concept is only relevant on the quantum scale and only makes physical sense when photons are treated as wave quanta - the minimum wave for any frequency being one cycle. Referring to a photon as a particle has rendered standard quantum theory incoherent and nonsensical; the particle interpretation is an unnecessary category error. As far as the theory-first approach goes you admit in the very next sentence that it has far more failures than successes (the complete list of undetectable things in MTP is much longer):

>Notably there has been spectacular failures with e.g. dark matter, dark energy, and quarks, but that is why we continue to dig for real solutions.

We both know the theory-first approach has failed spectacularly yet you seem to be suggesting that there is no alternative but to continue using the same method. At root that is what I don't understand about your position. If banging your head against the wall is giving you a headache then the best solution is to stop banging your head against the wall, isn't it?

**Comment by genotics** [comment]
When is a force not a force? When it's a pressure.

Perhaps this is an opportune moment to invite comment on two papers of a
total 4 at which I have been plugging away ever since AI came on the scene.
The papers are motivated by a long standing conviction that the ether
theory of my long deceased friend Adolphe Martin might somehow find
vindication and a broader range of application through a convergence with
the work of our colleague Amitabha Ghosh, thereby underpinning Amitabha's
own theory by elevating it from its phenomenological form to place it on a
material footing. The two papers I have just uploaded to my academia account

 <https://independent.academia.edu/CRoyKeys>

are an attempt, heavily saturated with AI wizardry, to put this conviction
into an articulate form suitable for scientific consumption. They are the
result of what seems like hundreds of hours and thousands of chats with 5
different AI models, each one checking the other's input. I invite comment
on these drafts, in which the answer to Sabina's conjecture might begin to
become clearer.
Two more draft papers are in preparation, one of which may be uploaded
today once I have polished a few more rough edges. It addresses the issues
of flat rotation curves, vertical acceleration and a topic close to
Martin's heart, the RAR, from the standpoint of Amitabha's theory of
velocity dependent inertial induction. Amitabha's approach to the flat
rotation curve issue has proved quite promising, but a detailed development
to address the whole of galactic dynamics turns out to be a daunting task.

Cheers, Roy

**Reply by HanDeBruijn** [reply]
I wrote in [thread 308](https://github.com/orgs/a-cosmology-group/discussions/308#discussioncomment-12779378) about AI.

**Reply by RedshiftDrift** [reply]
@genotics 
> chats with 5 different AI models, each one checking the other's input

AI models are not able to check anything, they can be made to agree with anything.  I tested this myself by successfully get ChatGPT to admit Earth is flat and the Big Bang never happened.  I'm sure I could get any LLM say that your papers are wrong, even if I haven't read them yet...
Sorry.

**Reply by budrap00** [reply]
@HanDeBruijn MTP = Modern Theoretical Physics. The term was capitalized in the paragraph preceding the acronym's usage.

**Reply by Francois-Zinserling** [reply]
Dear @genotics 
It has taken me too long to have looked at your link and I now sincerely regret not having done it earlier. I have thus far only managed a cursory scan through the first paper (Martin Cosmon Theory) and I am already impressed. This resonates well with me. Well, except for the airtime given to quarks but let's leave that for later. There are a few things I don't understand well yet, but then I am not fluent in gas theory so there's another hurdle to overcome.

I have come across several attempts to describe the aether as a gaseous (or also fluid or superfluid) substance before, but none with so many relevant conclusions as this one. In my model the aether is a photonic gas - iow it flows in all directions at 'c'. It compares with the author's 'average c' and is thus somewhat compatible.

I have yet to read it more in-depth, and also to overcome the differences in units used, but here are but a few points of overlap and agreement:
 - Invariant space. Thank you!
 -- not only is this compliant with SR, it gives an intuitive understanding of SR and is not plagued by paradoxes.
 - Similarities between electromagnetism and gas theory. I did not know so many existed.
 -- To elaborate on this, my conclusion of Newton's equation is rewritten as:
 
$F=μc^2/4πr^2*Mm$    (1)
which gives $μ=4πG/c^2$ which is also Kappa/2 in Einstein terms.

But note how this compares to equations for charge interactions! 
Because 

$F=k*QQ/r^2$

 can be rewritten as 

$F=μc^2/4πr^2*QQ$

not the same μ as in (1) of course ... just me struggling writing equations in here.

And the $1/4πr^2$ component describes a simple Gaussian sphere, again similar to electrodynamics, so no more arguing that gravity does not recede as to $1/r^2$. 

Both these conclusions are in agreement with Adolphe Martin.

**Reply by genotics** [reply]
I am pleased that you have found Adolphe's work so stimulating. Of course it is not surprising that there is a strong kinship between electromagnetism and fluid mechanics, as Maxwell contributed important components to the kinetic theory of gases, including the Maxwell-Boltzmann distribution and Maxwell's thermodynamic relations. It should not be surprising that the Lorentz factor actually exhibits the same form as the Prandtl-Glauert factor in aerodynamics, where the c parameter is the speed of sound rather than light. I am working on a further development of Adolphe's work and attempting to demonstrate that it also gives rise to Ghosh's velocity dependent inertial induction. Cheers, Roy
